docs/overview.asciidoc

```diff
@@ -0,0 +1,19 @@
+[id="{p}-overview"]
+== Overview
+
+Built on the Kubernetes Operator pattern, Elastic Cloud on Kubernetes (ECK) extends the basic Kubernetes orchestration capabilities to support the setup and management of Elasticsearch and Kibana on Kubernetes.
+
+With Elastic Cloud on Kubernetes you can streamline all those critical operations, such as:
+
+. Managing and monitoring multiple clusters
+. Scaling cluster capacity up and down
+. Changing cluster configuration
+. Dynamically scaling local storage (includes Elastic Local Volume, a local storage driver)
```

Comment:
Let's remove that one for now. We discussed how we should probably not advertise it too much until we commit to make it production-ready.

Reply:
Done.